recipe/build.sh

```diff
@@ -21,4 +21,5 @@ fi
 ./configure --with-pic --prefix=$PREFIX --disable-dependency-tracking
 make
 make check
+ln README.md README  # hack; snappy autotools build is broken in 1.1.6
```

Comment:
Guessing it has something to do with commit ( https://github.com/google/snappy/commit/7d7a8ec805192a192dda1906280387b2c8665225 ).